src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement/Application/UkrlpData/ProviderQueryResponse.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace SFA.DAS.RoatpCourseManagement.Application.UkrlpData
+{
+    [Serializable]
+    public class MatchingProviderRecords
```

Comment:
Suggest names to be changed to ProviderRecord or simply to Provider

Reply:
yes, done